spring-integration-sftp/src/test/java/org/springframework/integration/sftp/dsl/SftpTests.java

```diff
@@ -117,12 +119,14 @@ public void testSftpInboundStreamFlow() throws Exception {
 		assertNotNull(message);
 		assertThat(message.getPayload(), instanceOf(InputStream.class));
 		assertThat(message.getHeaders().get(FileHeaders.REMOTE_FILE), isOneOf(" sftpSource1.txt", "sftpSource2.txt"));
+		StreamUtils.copy((InputStream) message.getPayload(), new ByteArrayOutputStream());
```

Comment:
Interestingly, windows did not say that the file deletion failed, just silently left the file in place, preventing the deletion of the directory.